packages/forma-36-website/src/components/DocFormatter.js

```diff
@@ -144,6 +146,15 @@ const DocFormatter = ({ frontmatter, dataFromReadme, children }) => {
       <main>
         {activeSection === 'design' && hasDesignDocs && designDocs}
         {activeSection === 'develop' && hasDevelopDocs && developDocs}
+        {data.status === 'alpha' && (
+          <Flex marginBottom="spacingM">
+            <Note noteType="warning">
+              This component is in an alpha state. Breaking changes to the API
```

Comment:
Note for alpha components. But this would mean, I would remove this info from README files. Does this makes sense?

Reply:
Do all alpha components have the note in the readme at the moment? If so maybe we can remove this. I'd use this wording though.

Reply:
I think I'd like to keep the note in the README somehow, so that it can be understood without reading the website.

Reply:
ok, I will roll back. Maybe at some point, we will get this data in a different way from README and generate it in the Note component on the website. I think it's a nice touch.